fastplotlib/graphics/linecollection.py

```diff
@@ -49,33 +48,30 @@ def __init__(self, data: List[np.ndarray], z_position: Union[List[float], float]
             else:
                 _cmap = cmap
 
-            self.collection.append(LineGraphic(d, _z, _size, _colors, _cmap))
-
-    def _reset_feature(self):
-        pass
-
-    @property
-    def indices(self) -> Any:
-        pass
-
-    @indices.setter
-    @abstractmethod
-    def indices(self, indices: Any):
-        pass
+            self.data.append(LineGraphic(d, _z, _size, _colors, _cmap))
 
     @property
     def features(self) -> List[str]:
-        pass
+        return ["colors", "data"]
 
     def _set_feature(self, feature: str, new_data: Any, indices: Any):
```

Comment:
I think that for the `_set_feature` method in each `Graphic` or `GraphicCollection` we can replace `Any` for the indices to the indices that are appropriate for that graphic type.